uber/automated_emails.py

```diff
@@ -8,6 +8,8 @@
 class AutomatedEmail:
     instances = OrderedDict()
 
```

Comment:
There were some places where we were special-casing the ``PrevSeasonSupporter`` model.  This is now genericized.